code/utils/roi_extraction.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import numpy.linalg as npl
+import nibabel as nib
+import itertools as itt
+import matplotlib.pyplot as plt
+
+def co2vox(coordinate,mm_to_vox):
+	#I find this will give me some decimal number, which is unreasonable
```

Comment:
You may use python docstring (e.g. """ comments """) instead of comments (e.g. # comments) immediately below method names.

What I mean is that we should not leave docstring style comment wherever we want, but the first line immediately after method names is an appropriate place to use docstring because python automatically recognizes them as comments.

At least that's my understanding.